packages/apptive_grid_core/lib/src/network/authentication/web_auth_enabler/configure_web.dart

```diff
@@ -12,7 +12,7 @@ Future<void> enableWebAuth(ApptiveGridOptions options) async {
   final url = window.location.href;
   if (url.contains('state=')) {
     final issuerUri = Uri.parse(
-      'https://iam.zweidenker.de/auth/realms/${options.environment.authRealm}',
+      '${options.environment.url}/auth/${options.authenticationOptions.authGroup}/authorize',
     );
     final issuer = await openid.Issuer.discover(issuerUri);
```

Comment:
This issuer should be the same as in the authenticator. So maybe it's worthe to extract it into a variable

Reply:
done